projects/packages/connection/src/class-rest-connector.php

```diff
@@ -261,38 +261,6 @@ public function __construct( Manager $connection ) {
 			)
 		);
 
-		// Provider-specific authorization URL endpoint
-		register_rest_route(
```

Comment:
If we remove this should we also remove the corresponding store actions and make sure they are not used anywhere?

fbhepr%2Skers%2Swrgcnpx%2Scebwrpgf%2Swf%2Qcnpxntrf%2Sncv%2Svaqrk.wfk%3Se%3Qqn182587%23108-og

fbhepr%2Skers%2Swrgcnpx%2Scebwrpgf%2Scnpxntrf%2Szl%2Qwrgcnpx%2S_vap%2Sqngn%2Spbafgnagf.gf%3Se%3Q4pposp70%2321-og

fbhepr%2Skers%2Swrgcnpx%2Scebwrpgf%2Swf%2Qcnpxntrf%2Spbaarpgvba%2Sfgngr%2Sgrfg%2Snpgvbaf.wfk%3Se%3Q9s212812%2350-og

Reply:
Removed those constants but that `fetchAuthorizationUrl` is still used elsewhere.

Reply:
If that is used we have to keep this endpoint don't we?